### PR TITLE
docs(transfer): fix error

### PR DESCRIPTION
**Pull request by Ryan2128**
update `transfer` demo to vue 3

### CODE REVIEW

**Comment by Ryan2128** [comment]
#475

**Comment by zazzaz** [comment]
@Ryan2128 Hi, can you please keep the change of new slot syntax, but revert to the previous option API instead of  composition api

**Comment by Ryan2128** [comment]
> @Ryan2128 Hi, can you please keep the change of new slot syntax, but revert to the previous option API instead of composition api

OK

**Comment by Ryan2128** [comment]
Ready for review. @zazzaz

File: packages/transfer/src/useCheck.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const useCheck = (props: TransferPanelProps, panelState: TransferPanelSta
       if (typeof props.filterMethod === 'function') {
         return props.filterMethod(panelState.query, item)
       } else {
-        const label = item[labelProp.value] || item[keyProp.value].toString()
+        const label = item[labelProp.value] || item[keyProp.value].toString && item[keyProp.value].toString() || item[keyProp.value]
```
Comment (JeremyWuuuuu) [inline packages/transfer/src/useCheck.ts:20]: Why not `item[keyProp.value].toString?.()`?

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
It looks good to me after you fix the comment I mentioned.

File: packages/transfer/src/useCheck.ts (hunk 1 above)
Comment (Ryan2128) [reply packages/transfer/src/useCheck.ts:20]: Because the `key` of `item` may be a `number`, the type of `number` does't have the method of `toString`.

Comment (Ryan2128) [reply packages/transfer/src/useCheck.ts:20]: I did something wrong. It was a demo that was wrong,  the reason is `item[keyProp.value]` might be `undefined`. I will correct it later.

**Comment by JeremyWuuuuu** [comment]
is this ready to get merged？

**Comment by Ryan2128** [comment]
> is this ready to get merged？

Yes.

**Pull request merged**